logstreams/src/test/java/io/zeebe/logstreams/util/LogStreamRule.java

```diff
@@ -127,8 +127,8 @@ public void stopLogStream() {
     }
   }
 
-  public void openLogStream() {
-    logStream = new SyncLogStream(builder.build());
+  private void openLogStream() {
+    logStream = SyncLogStream.builder(builder).build(actorSchedulerRule.get());
```

Comment:
Could we change this to `SyncLogStream.builder(builder).withActorScheduler(actorSchedulerRule.get()).build();`